test/integration/krane_test.rb

```diff
@@ -16,8 +16,24 @@ def test_restart_black_box
     refute(fetch_restarted_at("redis"), "RESTARTED_AT env is present")
   end
 
+  def test_run_black_box
+    assert_empty(task_runner_pods)
+    assert_deploy_success(deploy_fixtures("hello-cloud", subset: ["template-runner.yml", "configmap-data.yml"]))
+    template = "hello-cloud-template-runner"
+    out, err, status = krane_black_box("run",
+      "#{@namespace} #{KubeclientHelper::TEST_CONTEXT} --template #{template} --command ls --arguments \"-a /\"")
+    assert_match("Successfully ran", err)
```

Comment:
Why does this match on `err` and not `out`? See the test above this one too.

Reply:
Curious why you tested for `Successfully ran`  over `Success`?

re stdout vs. stderr: It has to do with the original purpose of both. I think the render task is the easiest one to talk about. The rendered templates to go stdout because its actually what the program produces while the status / phase messages go to stderr since they're more metadata. https://www.jstorimer.com/blogs/workingwithcode/7766119-when-to-use-stderr-instead-of-stdout is a blog that does a better job than me at explaining it.

Reply:
> Curious why you tested for Successfully ran over Success?

No particular reason, really, other than 'Successfully ran' appears in the output. I can change it for consistency if you like!

> re stdout vs. stderr...

Ooo word, that makes good sense. Duly noted.

Reply:
I think consistency would be good.